analyses/driver/matcher.go

```diff
@@ -16,12 +20,39 @@ type PostResponseType struct {
 	ID          string `json:"id,omitempty"`
 }
 
+// VulnerabilitiesType is a Vulnerability Response structure
+type VulnerabilitiesType struct {
+	CveID    []interface{} `json:"cve_id"`
+	Cvss     int           `json:"cvss"`
+	CvssV3   string        `json:"cvss_v3"`
+	Cwes     []interface{} `json:"cwes"`
+	ID       string        `json:"id"`
+	Severity string        `json:"severity"`
+	Title    string        `json:"title"`
+	URL      url.URL       `json:"url"`
+}
+
+// AnalysedDepsType is type for Analysed Deps API Response
+type AnalysedDepsType struct {
+	Dependencies           []interface{}         `json:"dependencies"`
+	Ecosystem              string                `json:"ecosystem"`
+	LatestVersion          string                `json:"latest_version"`
+	Licenses               []interface{}         `json:"licenses"`
+	Name                   string                `json:"name"`
+	PrivateVulnerabilities []VulnerabilitiesType `json:"private_vulnerabilities"`
+	PublicVulnerabilities  []VulnerabilitiesType `json:"public_vulnerabilities"`
+	RecommendedVersion     string                `json:"recommended_version"`
+	Version                string                `json:"version"`
+	VulnerableDependencies []interface{}         `json:"vulnerable_dependencies"`
+}
+
 // GetResponseType is a argtype of RequestServer func
 type GetResponseType struct {
-	AnalysedDeps    []interface{}          `json:"analyzed_dependencies"`
-	Ecosystem       string                 `json:"ecosystem"`
-	Recommendation  map[string]interface{} `json:"recommendation"`
-	LicenseAnalyses map[string]interface{} `json:"license_analysis"`
+	AnalysedDeps        []AnalysedDepsType     `json:"analyzed_dependencies"`
+	Ecosystem           string                 `json:"ecosystem"`
+	Recommendation      map[string]interface{} `json:"recommendation"`
+	LicenseAnalyses     map[string]interface{} `json:"license_analysis"`
+	UnknownDependencies map[string]interface{} `json:"unknown_dependencies"`
```

Comment:
I guess I had asked to remove the unused field (which are typed as generic interface). Is there a reason for keeping this?

Reply:
UnknownDependencies would be used by coming up `verbose` flag

Reply:
Ack.

Reply:
Removed all unused items from this struct